src/main/commandServer/__tests__/httpCommandServer.spec.ts

```diff
@@ -47,16 +47,27 @@ describe(HttpCommandServer, () => {
    * or by developers during a typical edit-test-fix cycle, these are edge cases we can ignore for now.
    */
   itNonWindows("should fail to run rdctl shell when server isn't running", async() => {
-    const listSettingsRejects = expect(() => spawnFile(rdctlPath, ['list-settings'], { stdio: 'pipe' })).rejects;
+    const listSettingsRejects = expect(() => spawnFile(rdctlPath,
+      [
+        'list-settings', '--user=user', '--password=puser', '--port=6107'
+      ], { stdio: 'pipe' })).rejects;
 
     await listSettingsRejects.toHaveProperty('stdout', '');
     await listSettingsRejects.toHaveProperty('stderr', expect.stringMatching(/Error.*\/v\d\/settings.*dial tcp.*connect: connection refused/));
 
-    const rejects = expect(() => spawnFile(rdctlPath, ['shell', 'echo', 'abc'], { stdio: 'pipe' })).rejects;
+    // More errors are possible after a factory reset (or on a pristine system)
+    try {
+      const result = await spawnFile(rdctlPath, ['shell', 'echo', 'abc'], { stdio: 'pipe' });
 
-    await rejects.toHaveProperty('stdout', '');
-    await rejects.toHaveProperty('stderr', expect.stringContaining("Either run 'rdctl start' or start the Rancher Desktop application first"));
-    await rejects.toHaveProperty('stderr', expect.stringMatching(/(?:The Rancher Desktop VM needs to be created)|(?:The Rancher Desktop VM needs to be in state "Running" in order to execute 'rdctl shell', but it is currently in state)/));
+      expect(result.stdout).toBe('This call to rdctl shell should have failed');
```

Comment:
```suggestion
      expect(result).toBe('This call to rdctl shell should have failed');
```
Or even,
```ts
const result = spawnFile(rdctlPath, ['shell', 'echo', 'abc'], { stdio: 'pipe' });
await expect(result).rejects.toHaveProperty('stderr');
try {
  await result;
} catch (err: any) {
  …
}
```
(By using `expect(result).rejects` we get a better error, though we still need to await upon the result to trigger the throw — we can't get at the rejected value directly.)

Reply:
Yes, that's better.